dabplus-on-air-processing.spec

```diff
@@ -11,7 +11,7 @@ Name:     dabplus-on-air-processing
 BuildRequires: make
 BuildRequires: systemd
 
-Requires: liquidsoap
+Requires: liquidsoap = ${_ls_version}
```

Comment:
> Maybe I could do something relative based on the libdir with %include <../%{name}/file>. Liquidsoap 1.3.1 is coming soon. If that's ok with you, I'll give relative <> includes a try when I update to it (I'll have a real test rig set up then).

That's OK. I guess you could also use relative paths within the `%include "file"` directive, which will be relative to the actual ls script (according to [Liquidsoap's scripting language doc](http://savonet.sourceforge.net/doc-svn/language.html)) I don't know about the actual liquidsoap file location policies, but I would like to see them either in an independent subdirectory of `/usr/libexec/` or `/usr/share/`.